### ISSUE TITLE
"Unsupported File Extension" on recursive dryrun

### ISSUE THREAD

**Issue by automatonic** [dylansmith/node-exif-renamer#2]
`exif-renamer new/ -r -d`

Yields:

```                    
/volume1/@appstore/Node.js/usr/lib/node_modules/exif-renamer/node_modules/q/q.js:126
                    throw e;
                          ^
Error: unsupported file extension
    at Object.exifRenamer.process (/volume1/@appstore/Node.js/usr/lib/node_modules/exif-renamer/lib/exif-renamer.js:184:29)
    at Object.exifRenamer.rename (/volume1/@appstore/Node.js/usr/lib/node_modules/exif-renamer/lib/exif-renamer.js:212:14)
    at Promise.apply (/volume1/@appstore/Node.js/usr/lib/node_modules/exif-renamer/node_modules/q/q.js:1078:26)
    at Promise.promise.promiseDispatch (/volume1/@appstore/Node.js/usr/lib/node_modules/exif-renamer/node_modules/q/q.js:741:41)
    at /volume1/@appstore/Node.js/usr/lib/node_modules/exif-renamer/node_modules/q/q.js:1304:14
    at flush (/volume1/@appstore/Node.js/usr/lib/node_modules/exif-renamer/node_modules/q/q.js:108:17)
    at process._tickDomainCallback (node.js:463:13)
    at Function.Module.runMain (module.js:499:11)
    at startup (node.js:119:16)
    at node.js:906:3
```

It very well may be a bad extension, or some such. My assumption was that, by default, the extensions considered would be "jpg" and "jpeg".  If there was a bad file, it may be nice to warn and continue. Halting entirely is cool too, but a console message listing which file was the culprit would be extremely helpful. (My ultimate target is a directory tree with ~30K files)

**Comment by dylansmith** [comment]
I'm hoping the new exif library (https://www.npmjs.com/package/exif-parser) will solve this issue.

**Comment by dylansmith** [comment]
Updates in the following branch: https://github.com/dylansmith/node-exif-renamer/tree/replace-exif-parser

**Comment by dylansmith** [comment]
This was also fixed in version 0.7.0 (https://github.com/dylansmith/node-exif-renamer/commit/a7f507f60129929b74dcca1044d354d1dc3c4faf).
